explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -76,29 +76,28 @@ object AladinCell extends ModelOptics {
       // flag to trigger centering. This is a bit brute force but
       // avoids us needing a ref to a Fn component
       .useStateViewWithReuse(false)
-      // GuideStar candidates (loading, list)
-      .useState((true, List.empty[GuideStarCandidate]))
```

Comment:
Why is this gone?it seems we'd need to store the state somewhere?

Reply:
The state is within the `useStreamBy` hook.